card/mod/05_standard/spec/set/type/html_spec.rb

```diff
@@ -13,7 +13,7 @@
     expect(render_card(:closed_content, type: 'Html', content: "<strong>Lions and Tigers</strong>")).to eq('')
   end
 
-  it "should render inclusions" do
+  it "should render nests" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.